frontend/src/components/explorer/gemBrowser/ReactionTable.vue

```diff
@@ -62,9 +62,9 @@
                 </template>
               </td>
               <td>
-                <template v-for="(RP, i) in r.compartment_str.split(' => ')">
+                <template v-for="(RP, i) in r.compartment_str.split(/&#8658;|&#8660;/)">
                   <template v-if="i !== 0">{{ r.reversible ? ' &#8660; ' : ' &#8658; ' }}</template>
-                  <template v-for="(compo, j) in RP.split(' + ')">
+                  <template v-for="(compo, j) in RP.split(' ,')">
```

Comment:
I find that it's a bit difficult to comprehend with the nested `v-for`. Personally I would prefer something like the following but I'm not sure if that's an improvement.

```html
<td v-html="reformatCompartmentStr(r.compartment_str)" />
```

```javascript
reformatCompartmentStr(compartmentStr) {
  // matches any sequence that is not whitespace
  return compartmentStr.replace(/[^\s]+/g, (w) => {
    if (w === ',') {
      return '+';
    }

    if (w.match(/(&#8658;|&#8660)/)) {
      return w;
    }

    return `<a href="/explore/${this.model.short_name}/gem-browser/compartment/${idfy(w)}" @click="handleRouterClick">${w}</a>`;
  });
},
```

Reply:
@nanjiangshu  @mihai-sysbio What's your preference regarding this? I think this function is cleaner but possibly slightly easier to read the code as it is right now inside the template

Reply:
Hi @inghylt. I think both are OK but prefer the use of a function :)

Reply:
Like @inghylt, I too have a preference for keeping as much of the html creation in the `<template>` section of the Vue component. This has benefits in terms of eg syntax highlighting.